src/goto-symex/goto_symex_state.h

```diff
@@ -96,8 +96,72 @@ struct framet
   }
 };
 
+/// Intermediary class for encoding state of goto-symex
```

Comment:
No, the "state of goto-symex" should be in `goto_symext`. This should by the program state at a particular point during symbolic execution.

Reply:
Ok thanks, that's a bit clearer to me now. Although the distinction between `goto_statet` and `goto_symex_statet` is still a bit fuzzy.

Reply:
A `goto_statet` is the part of a state that can vary across different branches. Or at least most of that (the L1 renaming could also vary, but isn't stored).